### PR TITLE
docs: fix misspelling across the tree

### PR DESCRIPTION
**Pull request by nashif**
Found a few annoying typos and figured I better run script and
fix anything it can find, here are the results...

### CODE REVIEW

**Review by jhedberg — APPROVED** [review]

**Review by carlescufi — APPROVED** [review]

**Review by tyrantlizard77 — APPROVED** [review]

**Review by mike-scott — APPROVED** [review]

**Review by dbkinder — APPROVED** [review]

**Review by vanwinkeljan — APPROVED** [review]

File: include/sched_priq.h [hunk 1]
```diff
@@ -49,7 +49,7 @@ struct k_thread *z_priq_rb_best(struct _priq_rb *pq);
  * to the original Zephyr scheduler.  RAM requirements are
  * comparatively high, but performance is very fast.  Won't work with
  * features like deadline scheduling which need large priority spaces
- * to represet their requirements.
+ * to represents their requirements.
```
Comment (MaureenHelm) [inline include/sched_priq.h]: represent

File: lib/updatehub/Kconfig [hunk 2]
```diff
@@ -79,7 +79,7 @@ config UPDATEHUB_SHELL
 	  Activate shell module that provides UpdateHub commands like
 
 config UPDATEHUB_DTLS
-	bool"Activite communication CoAPS/DTLS"
+	bool"Activities communication CoAPS/DTLS"
```
Comment (MaureenHelm) [inline lib/updatehub/Kconfig]: Activate

File: scripts/filter-known-issues.py [hunk 3]
```diff
@@ -3,7 +3,7 @@
 Filters a file, classifying output in errors, warnings and discarding
 the rest.
 
-Given a set of regular expresions read from files named *.conf in the
+Given a set of regular expression read from files named *.conf in the
```
Comment (MaureenHelm) [inline scripts/filter-known-issues.py]: expressions?

File: soc/posix/inf_clock/soc.c [hunk 4]
```diff
@@ -5,7 +5,7 @@
  */
 
 /**
- * For all purposes, Zephyr threads see a CPU running at an infinitly high
+ * For all purposes, Zephyr threads see a CPU running at an infinity high
```
Comment (MaureenHelm) [inline soc/posix/inf_clock/soc.c]: infinitely

File: subsys/net/ip/tcp.c [hunk 5]
```diff
@@ -1101,7 +1101,7 @@ bool net_tcp_ack_received(struct net_context *ctx, u32_t ack)
 		/* Last sequence number in this packet. */
 		last_seq = sys_get_be32(tcp_hdr->seq) + seq_len - 1;
 
-		/* Ack number should be strictly greater to acknowleged numbers
+		/* Ack number should be strictly greater to acknowledge numbers
```
Comment (MaureenHelm) [inline subsys/net/ip/tcp.c]: acknowledged?

File: tests/kernel/critical/src/main.c [hunk 6]
```diff
@@ -154,7 +154,7 @@ void regression_thread(void *arg1, void *arg2, void *arg3)
 	ARG_UNUSED(arg2);
 	ARG_UNUSED(arg3);
 
-	k_sem_give(&ALT_SEM);   /* Activate alternate_tast() */
+	k_sem_give(&ALT_SEM);   /* Activate alternate_taste() */
```
Comment (MaureenHelm) [inline tests/kernel/critical/src/main.c]: alternate_test

**Review by MaureenHelm — CHANGES_REQUESTED** [review]

File: tests/kernel/critical/src/main.c (hunk 6 above)
Comment (nashif) [reply tests/kernel/critical/src/main.c]: alternate_facts() :-)

File: subsys/net/l2/ethernet/ethernet.c [hunk 7]
```diff
@@ -598,7 +598,7 @@ static int ethernet_send(struct net_if *iface, struct net_pkt *pkt)
 	}
 
 	/* If the ll dst addr has not been set before, let's assume
-	 * temporarly it's a broadcast one. When filling the header,
+	 * temporary it's a broadcast one. When filling the header,
```
Comment (ulfalizer) [inline subsys/net/l2/ethernet/ethernet.c]: *temporarily?

**Review by ulfalizer — APPROVED** [review]

File: arch/x86/gen_mmu_x86.py [hunk 8]
```diff
@@ -245,7 +245,7 @@ def populate_required_structs(self):
 
             # In-case the start address aligns with a page table entry other
             # than zero and the mem_size is greater than (1024*4096) i.e 4MB
-            # in case where it overflows the currenty PDE's range then limit the
+            # in case where it overflows the currently PDE's range then limit the
```
Comment (ulfalizer) [inline arch/x86/gen_mmu_x86.py]: *current

**Review by ioannisg — APPROVED** [review]

**Review by nvlsianpu — APPROVED** [review]

**Review by MaureenHelm — APPROVED** [review]

**Pull request merged**